src/index.html

```diff
@@ -4,14 +4,177 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">Surname:
+          <input 
+          type="text" 
+          name="surname"
+          id="surname"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">Name:
+          <input 
+          type="text" 
+          name="name" 
+          id="name"
+          min="1" 
+          max="100"  
+          autocomplete="off"
+          >
+        </label>
+        
+        <label class="form-field">How old are you?
+          <input 
+          type="number" 
+          name="age"
+          value="12" 
+          id="age"  
+          >
+        </label>
+
+        <label class="form-field">Full date of birth:
+          <input 
+          type="date" 
+          name="birth-date" 
+          id="birth"
+          >
+        </label>
+
+        <label class="form-field">I accept the term of the agreement:
+          <input 
+          type="checkbox" 
+          name="agreement"
+           id="agreement"
+          >
+        </label> 
+
+
+
+
+      </fieldset>
```

Comment:
```suggestion
        </label> 
      </fieldset>
```
Don't leave empty lines between parent and child elements